grails-app/domain/org/pih/warehouse/shipping/Shipment.groovy

```diff
@@ -697,5 +697,63 @@ class Shipment implements Comparable, Serializable {
     Boolean hasChildContainer() {
         shipmentItems?.any { it?.container }
     }
+
+    Set<Event> getCustomEvents() {
+        return events.findAll { it.eventType?.eventCode?.name()?.startsWith("CUSTOMS") }
```

Comment:
As discussed on slack, please add  `listSystemEventTypeCodes` and `listCustomEventTypeCodes` to the enum, and use the `listCustomEventTypeCodes` here

Reply:
added